elyra/pipeline/processor_airflow.py

```diff
@@ -216,6 +221,27 @@ def _cc_pipeline(self, pipeline, pipeline_name):
                                                           operation)
 
             else:
+                # Change value of variables according to their type. String variables must include
+                # quotation marks in order to render properly in the jinja template and dictionary
+                # values must be converted from strings.
+                component = self._component_registry.get_component(operation.classifier)
+                for component_property in component.properties:
+                    if component_property.ref in ['runtime_image', 'component_source', 'component_source_type']:
```

Comment:
Nit: not future proof.

Reply:
we will refactor the generic components in a future pr